packages/unchained-client/src/evm/parser/arbitrumBridge.ts

```diff
@@ -140,6 +140,17 @@ export class Parser implements SubParser<Tx> {
               },
             })
           }
+          case this.l2ArbitrumGatewayAbi.getFunction('finalizeInboundTransfer')!.selector:
```

Comment:
Can we opt chain this guy?

```suggestion
          case this.l2ArbitrumGatewayAbi.getFunction('finalizeInboundTransfer')?.selector:
```

Reply:
Ideally we do the already-in-develop one above, too: https://github.com/shapeshift/web/pull/7647/files#diff-fc2bdb4c69345bec1c8fc1b03bbcae66368cc0a318d6c4ab96ed4bef0ddc5c9eR124

Reply:
Was following the pattern above but prefer non-null assert in this case since it's just bad typings, the ABI *is* deterministic and guaranteed to have this guy 🙏🏽